### ISSUE TITLE
Einheitliche Toolbars

### ISSUE THREAD

**Issue by sheepyhollow** [zentrumnawi/solid-frontend#1401]
Im Moment sind alle Toolbars sehr unterschiedlich:

Quiz: keine Toolbar
![image](https://user-images.githubusercontent.com/13869236/154349978-6630fb98-5c89-46b8-b9b0-82f2d25ba509.png)

Profiles: graue und sehr hohe Toolbar
![image](https://user-images.githubusercontent.com/13869236/154350180-c3022217-92ee-4a0e-a58d-f46f3ff09555.png)

Slideshow: weiße Toolbar
![image](https://user-images.githubusercontent.com/13869236/154350319-64c7674b-40e1-4f29-94aa-ad6dbc95ea8e.png)

Das sollte in der App einheitlicher aussehen - der Standard soll die Slideshow-Toolbar sein, das sieht aktuell am besten aus.

## Profiles
* Toolbar weiß (nicht secondary color) und mit einem leichten Schatten.
* Wenn möglich, könnte die Toolbar weniger hoch sein (siehe auch #1375).
* Back-Button (links) mit dem auf die landing page zurückgeführt wird - wenn das nicht doof aussieht.
* alles andere kann bleiben

## Quiz
* Neue Toolbar im gleichen Styling wie Slideshows (gibt es Vorschläge zum "kleinen Bildschirm"?)
* Back-Button: Wenn gerade eine Quiz-Session läuft, soll ein Popup erscheinen das darauf hinweist, dass das Quiz abgebrochen wird.
* in der Toolbar wird das Tag gezeigt und die Anzahl der Fragen (z.B. "Frage 5 von 10").
* Close-Button: die Auswertung wird gezeigt
* der Fortschritt soll identisch zur Slideshow mit einer `progress-bar` gemacht werden.
* die Quiz-Cards bleiben unverändert (zum Styling machen wir evtl. ein eigenes Issue)
* der `Lösungen anzeigen`-Button bleibt unverändert

**Comment by kerstinhartmann** [comment]
Was ist aus deiner Idee mit dem sekundär-farblichen "Schmuckstreifen" geworden? 
Dieser würde oberhalb der Toolbar angezeigt.

Auf kleinen Bildschirmen im Prinzip genauso wie auf großen Bildschirmen oder? 
links der Zurück-Pfeil, mittig "Tag" und darunter + kleiner "Frage 2 von 5" sowie rechts das x zum Quiz (vorzeitig) beenden. 
--> Dieser Button würde dann noch **von der Quiz-Card in die Toolbar verschoben**. Oder ist die Toolbar bei längeren Tag-Titeln dann zu voll?

**Comment by sheepyhollow** [comment]
Nein, das könnten wir so machen - das ist ja potenziell zweizeilig. Den Schmuckstreifen hatte ich noch rausgelassen, um das Issue nicht noch aufzublasen 😅 Aber - bittesehr:

![image](https://user-images.githubusercontent.com/13869236/154525744-7bb5d81d-6271-4f78-8b37-e0f46428fe1d.png)

(hier fehlt nur der Schatten am unteren Rand weil Mockup)

**Comment by sheepyhollow** [comment]
Ich weiß nicht, ob uns die [Material Guidelines](https://material.io/design/navigation/understanding-navigation.html#reverse-navigation) weiterbringen, aber es wäre schon gut, eine einheitliche Navigations-Strategie zu haben.

**Comment by sheepyhollow** [comment]
Ich denke, das hier ist auch gelöst.
